tests/test_embit_utils.py

```diff
@@ -373,7 +373,6 @@ def test_parse_derivation_path():
         (None, SC.CUSTOM_DERIVATION, False, 5): "m/123'/9083270/9083270/9083270/9083270/0/5",
 
         # non-standard change and/or index
-        (None, SC.CUSTOM_DERIVATION, None, 5): "m/9'/78/5",
```

Comment:
Note that this line is not a duplicate of the following line (78 here is unhardened but it is hardened in the following line).

Reply:
this is a dictionary... with `key: value` entries... the value is different, but as the key is the same it is overwritten... that how I understood what the linter tries to say here.

Reply:
Ah, oof, that was messy of me.

Can you preserve that line but just comment it out for now? Ideally also with a `# TODO` note.